students/kyeongrae-kim/user/views.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
+from user.models            import User
+import re
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        users_info = User.objects
+        regex = '^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$'
+
+        try:
+            if not (re.search(regex, data['email'])):                   
+                return JsonResponse({'SIGNUP':'BAD_REQUEST_KEY_ERROR'}, status=400)
```

Comment:
```suggestion
                return JsonResponse({'MESSAGE':'KEY_ERROR'}, status=400)
```
키를 통일 시킨 것은 잘 하셨습니다. 이미 내가 요청한 곳을 알기 때문에 어떤 기능에서 난 에러인지 명시 안 하셔도 되며, 같은 에러여도 기능마다 key가 달라지게 됩니다. 그리고 이미 400으로 bad request인 것까지는 알기 때문에 bad request도 없어도 될 것 같습니다.

Reply:
메세지의 불필요한 중복을 줄이고 정확한 의미 전달을 위해 미사여구를 줄였습니다.